osu.Game/Overlays/Settings/Sections/Graphics/MainMenuSettings.cs

```diff
@@ -2,25 +2,55 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Allocation;
+using osu.Framework.Graphics;
 using osu.Game.Configuration;
+using osu.Game.Online.API;
+using osu.Game.Users;
 
 namespace osu.Game.Overlays.Settings.Sections.Graphics
 {
     public class MainMenuSettings : SettingsSubsection
     {
         protected override string Header => "User Interface";
 
+        private SettingsEnumDropdown<MainMenuBackgroundMode> backgroundDropdown;
+
         [BackgroundDependencyLoader]
-        private void load(OsuConfigManager config)
+        private void load(IAPIProvider api, OsuConfigManager config)
         {
-            Children = new[]
+            Children = new Drawable[]
             {
                 new SettingsCheckbox
                 {
                     LabelText = "Parallax",
                     Bindable = config.GetBindable<bool>(OsuSetting.MenuParallax)
                 },
+                backgroundDropdown = new SettingsEnumDropdown<MainMenuBackgroundMode>
+                {
+                    LabelText = "Main menu background",
+                    Bindable = config.GetBindable<MainMenuBackgroundMode>(OsuSetting.MenuBackgroundMode)
+                }
             };
+
+            api.LocalUser.BindValueChanged(user => onUserChanged(user.NewValue), true);
```

Comment:
```suggestion
            api.LocalUser.BindValueChanged(onUserChanged, true);
```

we prefer the `valuechanged` event to be handled at the destination.